tensorboard/components/tensor_widget/tensor-widget.ts

```diff
@@ -0,0 +1,34 @@
+
+/* Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the 'License');
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an 'AS IS' BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+import {SingleTensorWidget, SingleTensorWidgetOptions, TensorView} from "./types";
```

Comment:
FYI: so, this is okay especially since you are writing reusable component that works outside of TensorBoard but if you want to import any of the existing TensorBoard module, you will have to mix namespace with modules.

Reply:
ACK. tensor-widget won't import anything from TensorBoard modules. Only the review importing will happen.